ComfortZoneApp/app/src/main/java/com/example/comfortzone/flight/data/IataClient.java

```diff
@@ -0,0 +1,107 @@
+package com.example.comfortzone.flight.data;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.example.comfortzone.BuildConfig;
+import com.example.comfortzone.flight.callbacks.IataCallback;
+import com.example.comfortzone.flight.models.AccessToken;
+import com.example.comfortzone.flight.models.Airport;
+import com.google.gson.Gson;
+
+import java.io.IOException;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.FormBody;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+public class IataClient extends OkHttpClient {
+
+    public static final String TAG = "IataClient";
+    public static final String AMADEUS_BASE_URL = "https://test.api.amadeus.com/v1/reference-data/locations/airports";
```

Comment:
just FYI, the base url is actually just `test.api.amadeus.com`, others are scheme, paths, queries, etc:  https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL